UserApi/UserApi/Controllers/UserController.cs

```diff
@@ -177,34 +178,35 @@ public async Task<IActionResult> GetUserByEmail(string email)
         }
 
         /// <summary>
-        ///     Get Ejudiciary Judges from AD
+        ///     DEPRECATED - Methods using this should use be replaced with the override version
+        ///     of this method that takes a username.
         /// </summary>
-        [HttpGet("ejudJudges", Name = "GetEjudiciaryJudges")]
-        [OpenApiOperation("GetEjudiciaryJudges")]
+        [HttpGet("judges", Name = "GetJudges")]
+        [OpenApiOperation("GetJudges")]
         [ProducesResponseType(typeof(List<UserResponse>), (int)HttpStatusCode.OK)]
         [ProducesResponseType((int)HttpStatusCode.NotFound)]
-        public async Task<IActionResult> GetEjudiciaryJudges()
+        public async Task<IActionResult> GetJudges()
         {
-            var ejudiciaryJudges = await _distributedCache.GetOrAddAsync(() => _userAccountService.GetEjudiciaryJudgesAsync());
-            
-            if (ejudiciaryJudges == null || !ejudiciaryJudges.Any())
+            var adJudges = await _distributedCache.GetOrAddAsync(() => _userAccountService.GetJudgesAsync());
+
+            if (adJudges == null || !adJudges.Any())
             {
                 return Ok(new List<UserResponse>());
             }
 
-            return Ok(ejudiciaryJudges);
+            return Ok(adJudges);
         }
 
         /// <summary>
-        ///     Get Judges from AD
+        ///     Gets a list of judges with the filtered username
         /// </summary>
-        [HttpGet("judges", Name = "GetJudges")]
-        [OpenApiOperation("GetJudges")]
+        [HttpGet("judgesbyusername", Name = "GetJudgesByUsername")]
```

Comment:
Just a thought, could we preserve the "judges" url here in place of "judgesbyusername"?

It should work I think - GET judges should call the old "GetJudges" endpoint, GET judges?username=test should call the new "GetJudgesByUsername" one.

Reply:
Hi Oliver. I tried that already and unfortunately it doesn't work due to the Nswag command failing. .NET is fine it is NSwag that is the issue.